src/runtime/pkg/katautils/config-settings.go.in

```diff
@@ -79,6 +79,7 @@ const defaultHotplugVFIOOnRootBus bool = false
 const defaultPCIeRootPort = 0
 const defaultEntropySource = "/dev/urandom"
 const defaultGuestHookPath string = ""
+const defaultGuestHookTimeout int32 = 10
```

Comment:
nit:  I would make this as unsigned value.

Reply:
but in OCI defined, it also used `i32`, I think we should keep it consistent.
https://github.com/kata-containers/kata-containers/blob/main/src/libs/oci/src/lib.rs#L188